cellmlmanip/_singularity_fixes.py

```diff
@@ -0,0 +1,252 @@
+"""
+``remove_fixable_singularities`` specifies a method to remove fixable singularities from a model
+"""
+from math import isclose
+
+import networkx as nx
+from sympy import (
+    Abs,
+    Add,
+    Eq,
+    Float,
+    I,
+    Mul,
+    Piecewise,
+    Pow,
+    Wild,
+    exp,
+    log,
+    solveset,
+)
+
+import cellmlmanip
+from cellmlmanip.model import FLOAT_PRECISION
+
+
+ONE = cellmlmanip.Quantity(1.0, 'dimensionless')
```

Comment:
Would this be better?
```suggestion
from .model import FLOAT_PRECISION, Quantity

ONE = Quantity(1.0, 'dimensionless')
```